src/common/Finisher.h

```diff
@@ -132,17 +134,18 @@ class Finisher {
 
   /// Construct an anonymous Finisher.
   /// Anonymous finishers do not log their queue length.
-  Finisher(CephContext *cct_) :
+  Finisher(CephContext *cct_, const char *tn) :
```

Comment:
Yeah; I'd change the type here to string and don't set a thread name on the anonymous variant.